tests/utils.py

```diff
@@ -125,3 +125,11 @@ def kill_process():
 
     finally:
         timer.cancel()
+
+
+def filter_match(match, start_method=None):
+    if sys.platform == "win32":
+        return "UNKNOWN"
+
+    if start_method == "forkserver" and sys.version_info < (3, 7):
+        return "UNKNOWN"
```

Comment:
don't you want to return `match` if none of the condition above are verified?

Reply:
Indeed! Thanks